tooltester/src/main/java/io/dockstore/tooltester/helper/GA4GHHelper.java

```diff
@@ -87,4 +91,47 @@ private static boolean matchVerifiedSource(List<String> filter, String verifiedS
         }
         return tools;
     }
+
+    /**
+     * Determines whether the runner and descriptor type is compatible with each other
+     *
+     * @param runner             The runners available ("cwltool", "cwl-runner", "cromwell"
+     * @param descriptorTypeEnum The descriptor type of the ToolVersion
+     * @return Whether the runner can be used for the descriptor type
+     */
+    static boolean runnerSupportsDescriptorType(String runner, ToolVersion.DescriptorTypeEnum descriptorTypeEnum) {
+        switch (runner) {
+        case "cwltool":
+        case "cwl-runner":
+            return descriptorTypeEnum == ToolVersion.DescriptorTypeEnum.CWL;
+        case "cromwell":
```

Comment:
Does the fact that we can run CWL with Cromwell matter here?

Reply:
Currently it's untested